src/api/routes/tokens/tokens.ts

```diff
@@ -0,0 +1,67 @@
+import { addAsync, RouterWithAsync } from '@awaitjs/express';
+import * as express from 'express';
+import { DataStore } from '../../../datastore/common';
+import {
+  FungibleTokenMetadataResponse,
+  NonFungibleTokenMetadataResponse,
+} from '@stacks/stacks-blockchain-api-types';
+
+export function createTokenRouter(db: DataStore): RouterWithAsync {
```

Comment:
This should include a route to enumerate the list of tokens, similar to `/extended/v1/tx` and `/extended/v1/block`.